backend/cdkproxymain.py

```diff
@@ -24,9 +22,6 @@
     f"Application started for envname= `{ENVNAME}` DH_DOCKER_VERSION:{os.environ.get('DH_DOCKER_VERSION')}"
 )
 
-load_modules(modes={ImportMode.CDK})
```

Comment:
was this unnecessary because the loading also happens [in app.py ](https://github.com/awslabs/aws-dataall/blob/main-v2/backend/dataall/base/cdkproxy/app.py#L16)?

Reply:
Yup exactly

Reply:
My understanding is, that in case the pipeline type is CDK Pipeline (the one managed with DDK), this app.py is not going to be used, as another one is created with the DDK commands. See [here](https://github.com/awslabs/aws-dataall/blob/main-v2/backend/dataall/base/cdkproxy/cdk_cli_wrapper.py#L116) `app_path` is overwritten and [here](https://github.com/awslabs/aws-dataall/blob/main-v2/backend/dataall/modules/datapipelines/cdk/datapipelines_cdk_pipeline.py#L142) the new app file is created.

In this case, aren't we going to missed the CDK modules, as this loading is not going to happen in the other app.py file?

Reply:
When the pipeline type is `cdkpipeline` we load the `DatapipelinesCdkCLIExtensionModuleInterface` which loads the `CDKPipeline()` class for us

The `CDKPipeline()` class initializes a new DDK App and CodeCommit Repo and calls `cdkdeploy` of that newly created `app.py`

For that DDK App to deploy there is no need to reference any additional data.all CDK module code - there just requires cdk and ddk pip libraries to be installed (as is the case in `requirements.txt`)

I do not think there should be any issue here with not having `load_modules({ImportMode.CDK})` anywhere but the `app.py`